pkg/api/v1/event.go

```diff
@@ -0,0 +1,49 @@
+package v1
```

Comment:
Since this object depends on `v1.AttachedObjectReference`, it must be in `pkg/api/v1` and not `pkg/api/v0`. If it were in the latter, an import cycle would be created.